showcase/src/app/demo-ui/components/table-with-custom-cell-rendering/table-with-custom-cell-rendering.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component } from "@angular/core";
+import { StarkTableColumnProperties, StarkTableFilter } from "@nationalbankbelgium/stark-ui";
+
+const DUMMY_DATA: object[] = [
+	{ id: 1, cost: 12, description: "number one", expendedInformation: "<span>More infos about the current row</span>" },
```

Comment:
"expendedInformation" is linked to your other PR about collapse. Could you please remove it ?

Reply:
done